experimental/packages/opentelemetry-instrumentation-grpc/README.md

```diff
@@ -46,6 +46,14 @@ gRPC instrumentation accepts the following configuration:
 | ------- | ---- | ----------- |
 | [`ignoreGrpcMethods`](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-instrumentation-grpc/src/types.ts#L25) | `IgnoreMatcher[]` | gRPC instrumentation will not trace any methods that match anything in this list. You may pass a string (case-insensitive match), a `RegExp` object, or a filter function. |
 
+## Semantic conventions for RPC spans -
```

Comment:
Did you verify that the semantic conventions are correctly followed?

Reply:
@dyladan I had not instrumented application with grpc. I had gone through document.

Reply:
@dyladan I have verified semantic conventions for metrics. I have added here results in repo here: https://github.com/siddhikhapare/opentelemetry-metrics-example/tree/main/results Please have a look once.